<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-8QFFSKO" failures="1" tests="1" name="verificationUsingTestNG.SoftAssesrStudy" time="0.049" errors="0" timestamp="2022-07-29T13:14:57 IST" skipped="0">
  <testcase classname="verificationUsingTestNG.SoftAssesrStudy" name="SoftAssertMethods" time="0.049">
    <failure message="The following asserts failed:
	value of A and c are not equals hence TC is failed expected [limkar] but found [Sulakshana],
	 A and B Both values are equal hence TC is failed did not expect [Sulakshana] but found [Sulakshana],
	given value of f is false hence TC is failed expected [true] but found [false],
	given value of e is true hence TC is failed expected [false] but found [true],
	given value of c is not null hence TC is failed expected [null] but found [limkar],
	given value of d  is null hence TC is failed expected object to not be null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
value of A and c are not equals hence TC is failed expected [limkar] but found [Sulakshana],
 A and B Both values are equal hence TC is failed did not expect [Sulakshana] but found [Sulakshana],
given value of f is false hence TC is failed expected [true] but found [false],
given value of e is true hence TC is failed expected [false] but found [true],
given value of c is not null hence TC is failed expected [null] but found [limkar],
given value of d  is null hence TC is failed expected object to not be null
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at verificationUsingTestNG.SoftAssesrStudy.SoftAssertMethods(SoftAssesrStudy.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- SoftAssertMethods -->
  <system-out/>
</testsuite> <!-- verificationUsingTestNG.SoftAssesrStudy -->
